codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/AsyncClientInterface.java

```diff
@@ -118,18 +121,18 @@ private MethodSpec builder() {
     private Iterable<MethodSpec> operationsAndSimpleMethods() {
         List<MethodSpec> methods = operations();
         methods.addAll(model.getOperations().values().stream()
-                            .map(this::simpleMethods)
+                            .map(this::addMethodOverloads)
                             .flatMap(List::stream)
                             .map(MethodSpec.Builder::build)
-                            .collect(Collectors.toList()));
-        return methods;
+                            .collect(toList()));
+        return methods.stream().sorted(Comparator.comparing(m -> m.name)).collect(toList());
```

Comment:
Questions: I saw some places we are doing static imports and some are not. Do we want to make it more consistent?